tools/internal/package.json

```diff
@@ -23,5 +23,8 @@
   "dependencies": {
     "yargs": "^16.0.0"
   },
-  "devDependencies": {}
+  "devDependencies": {
+    "@itwin/build-tools": "workspace:*",
```

Comment:
What's the reason for the `@itwin/build-tools` dep?

Reply:
That's for access to the BentleyMochaReporter.